### DISCUSSION TITLE
Welcome to pan-os-ansible Discussions!

### DISCUSSION THREAD

**Discussion by shinmog** [PaloAltoNetworks/pan-os-ansible#203] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by david-johnstone** [comment]
Hi guys, what is the correct and polite way to request a bug issue to be reopened? In particular: https://github.com/PaloAltoNetworks/pan-os-ansible/issues/324

Many thanks.

**Comment by sikulkar** [comment]
Hi, 

I am looking into the documentation for panos_import and I noticed that the pan-os-ansible module does not mention that it supports "private-key" as an import category. The closest option is keypair. 

Going through the API browser, I do see keypair as well as private-key options available doing the same thing? Just wondering if what's the difference between these 2 options ?
